include/base/NekInterface.h

```diff
@@ -301,6 +301,22 @@ void volumeSolution(const int order, const bool needs_interpolation, const field
  */
  void flux(const int elem_id, const int order, double * flux_face);
 
+/**
+ * Send velocity from SAM to the nekRS mesh
+ * @param[in] elem_id global element ID
+ * @param[in] order enumeration of the surface mesh order (0 = first, 1 = second, etc.)
+ * @param[in] velocity_SAM is SAM boundary velocity
+ */
+ void velocity(const int elem_id, const int order, const double* velocity_SAM);
```

Comment:
If the temperature/velocity we're sending in are just a single double value, we don't need to use a pointer here. You can change this to `const double velocity`). Let's also remove the explicit reference to SAM here. There might be a few other places I missed as well.

Reply:
My fault I'll doublecheck everywhere for SAM